StPetersburg/IVEf17.xml

```diff
@@ -0,0 +1,301 @@
+                      <title type="complete" ref="LIT2509Weddas"/>
+                      <textLang mainLang="gez"/>
+                      <note>Starts from Monday.</note>
+                    </msItem>
+                    <msItem xml:id="ms_i1.5">
+                      <locus from="129r" to="134v"/>
+                      <title type="complete" ref="LIT2509Weddas"/>
+                      <textLang mainLang="gez"/>
+                      <note>Starts from Monday.</note>
+                    </msItem>
+
+                    <msItem xml:id="ms_i1.6">
+                      <locus from="137v" to="142v"/>
+                      <title type="complete" ref="LIT2807RepCh91"/>
+                      <textLang mainLang="gez"/>
+                      <note> Excerpt from the
+                       Maṣḥafa saʿātāt (salām-praises for the Saviour and
+                       St. Mary for the 1st, 3rd and 9th hour).</note>
+                    </msItem>
+
+
+               </msContents>
+
+               <!--  numbers of folios are not certain -->
+
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf">143</measure>
+                           <dimensions type="outer" unit="mm">
+                              <height>25</height>
+                              <width>15</width>
+                              <depth/>
+                           </dimensions>
+                        </extent>
+                        <condition key="good"/>
+                     </supportDesc>
+
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="32 33">
+                           <locus from="3r" to="93v"/>
+                           <note/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <date notBefore="1700" notAfter="1799"/>
+                        <desc>
+                           <persName ref="PRS9602Turaev"/>
+                           describes the script as "large" and "nice".
+                 </desc>
+                     </handNote>
+                  </handDesc>
+
+                  <decoDesc>
+                     <decoNote type="drawing" xml:id="d1">
+                        <locus target="#1r"/>
+                        <desc>Uncoloured depiction of St. George of
+                        Lydda and the donor. Legend above reads <q>ደራጐን</q>,
+                        legend below <q>ደራጎን</q>.
+                        A female figure with rosary stands on the left,
+                        the caption reads <q>ዘከመ᎓ ተማኅፀነ᎓ ወለተ᎓ ገብርኤል</q>.
```

Comment:
see the mark up of legends and captions in the guidelines. I think what you described here as caption is called legend there and vice versa?